### PR TITLE
[8.x] Remove redirecrUri from "required" config

### PR DESCRIPTION
**Pull request by Nyholm**
There are many many API endpoint you can call that do not require the `redirectUri`. I don't think it should be a requirement to creating a `Management` object.

### CODE REVIEW

**Review by evansims — APPROVED** [review]
LGMT 👍 I fixed up a few problem spots that opened up by removing that as a required var, and fixed up the test.

File: src/API/Authentication.php [hunk 1]
```diff
@@ -80,14 +80,20 @@ public function getHttpClient(): HttpClient
     public function getAuthorizationLink(
         ?array $params = null
     ): string {
-        $params = Shortcut::mergeArrays([
+        $returnUri = $this->configuration->getRedirectUri();
```
Comment (Nyholm) [inline src/API/Authentication.php:83]: The redirectUri could also be part of $param, right?

**Review by Nyholm — COMMENTED** [review]
Thank you for merging and fixing this. I just have one question that applied to all the endpoints requiring redirect_uri

File: src/API/Authentication.php (hunk 1 above)
Comment (evansims) [reply src/API/Authentication.php:83]: Good catch, thanks! Fixed w/ a separate commit to the 8.x branch.

**Pull request merged**